licenses/management/commands/publish.py

```diff
@@ -0,0 +1,148 @@
+import os
+import subprocess
+from argparse import ArgumentParser
+
+from django.core.management import BaseCommand
+
+from cc_licenses.settings.base import TRANSLATION_REPOSITORY_DIRECTORY
+
+# Go to cc-licenses-data and do something
+GO_TO_TRANSLATIONS_REPO = f"cd {TRANSLATION_REPOSITORY_DIRECTORY} && "
+PULL_BRANCHES_DOWN_FROM_DEVELOP = "git checkout develop && git pull"
+PULL_DOWN_TRANSLATION_BRANCHES = (
+    GO_TO_TRANSLATIONS_REPO + PULL_BRANCHES_DOWN_FROM_DEVELOP
+)
+
+
+def run_django_distill():
+    """Outputs the build directory
+    """
+    my_env = os.environ.copy()
+    my_env["DJANGO_SETTINGS_MODULE"] = "cc_licenses.settings.publish"
+    cmd = "python manage.py distill-local --quiet --force"
+    return subprocess.run(
+        cmd, shell=True, check=True, text=True, input="YES", env=my_env,
+    )
+
+
+def git_branch_status(branch: str):
+    """Checks if there is a build to commit using git status
+
+    Returns True or False
+    """
+    check_branch_status_cmd = (
+        GO_TO_TRANSLATIONS_REPO + f"git checkout {branch} && " + "git status"
+    )
+    status = (
+        subprocess.check_output(check_branch_status_cmd, shell=True)
+        .decode()
+        .split("\n")
+    )
+    if "nothing to commit, working tree clean" in status:
+        return False
+    return True
+
+
+def git_on_branch_and_pull(branch: str):
+    """Returns a string represention of the git command to checkout and pull from a branch"""
+    git_on_branch_and_pull_cmd = GO_TO_TRANSLATIONS_REPO + (
+        f"git checkout {branch} && git pull origin {branch}"
+    )
+    return subprocess.run(git_on_branch_and_pull_cmd, shell=True, check=True)
+
+
+def git_commit_and_push(branch: str):
+    """Returns a string representation of the git command to checkout, commit, and push branch"""
+    commit_and_push_cmd = GO_TO_TRANSLATIONS_REPO + (
+        "git add build/ && "
+        f"git checkout {branch} && "
+        f"git commit -m '{branch}' && "
+        f"git push origin {branch}"
+    )
+    return subprocess.run(commit_and_push_cmd, shell=True, check=True,)
+
+
+def pull_translations_branches():
+    """Git pulls branches in cc-licenses-data to update local git registry"""
+    return subprocess.run(PULL_DOWN_TRANSLATION_BRANCHES, shell=True, check=True,)
+
+
+def list_open_branches():
+    """List of open branches in cc-licenses-data repo
+    """
+    list_branches_cmd = GO_TO_TRANSLATIONS_REPO + "git branch --list"
+    pull_translations_branches()
+    branches = (
+        subprocess.check_output(list_branches_cmd, shell=True).decode().split("\n")
```

Comment:
`str` has a `splitlines()` method that works like `split("\n")` but handles more cases of line endings so it's a little more universal. (I can never remember whether subprocess output includes "\n" or "\r\n" but splitlines works either way.)